.vscode/c_cpp_properties.json

```diff
@@ -1,12 +1,16 @@
 {
-  "configurations": [
-    {
-      "name": "FreeCAD",
-      "includePath": ["${workspaceFolder}/**"],
-      "cStandard": "c17",
-      "cppStandard": "c++17",
-      "configurationProvider": "ms-vscode.cmake-tools"
-    }
-  ],
-  "version": 4
-}
+    "configurations": [
+        {
+            "name": "FreeCAD",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/usr/include/QtCore",
+                "/usr/include/python3.11"
+            ],
+            "cStandard": "c17",
+            "cppStandard": "c++17",
+            "configurationProvider": "ms-vscode.cmake-tools"
+        }
+    ],
+    "version": 4
+}
```

Comment:
I don't think you should touch these files unless you know what youa re doing and actually want to change the default for everyone

Reply:
Afaik this just adds the two paths so that vscode can find the proper include headers (on linux of course). I've intentionally named the commit [vscode stuff: drop or merge?](https://github.com/FreeCAD/FreeCAD/pull/9175/commits/cf4651588916d857454ca017a1a4cc5f50703a37), so I'm perfectly fine reverting these changes before merge

Reply:
@adrianinsaval we should probably remove this folder itself and add it to `.gitignore`

Reply:
It was added recently by #9098

Reply:
adding to gitignore would be a good idea, when you actually want to make changes there you can still explicitly add them to the commit. Here that commit should be dropped IMO

Reply:
I agree the commit is not really part of this PR. Still these changes made vscode happy on both of my machines (arch and fedora). Not trying to force this or anything, its just my opinion since this has been useful to me. Unless someone is favourable on keeping this I will drop it before merge.

Reply:
It probably doesn't make sense to have the include path changes in there, since they are system-specific. The rest of the changes are fine (though I agree that we should add the files to .gitignore).